extension/src/popup/constants/metricsNames.ts

```diff
@@ -84,7 +84,10 @@ export const METRIC_NAMES = {
 
   accountScreenAddAccount: "account screen: created new account",
   accountScreenCopyPublickKey: "account screen: copied public key",
-  accountScreenImportAccount: "account screen: created new account",
+  accountScreenImportAccount: "account screen: imported new account",
```

Comment:
pretty sure this should have been "imported" instead of "created" before, but we need the new metric now anyways